src/commands/notify.rs

```diff
@@ -267,6 +267,10 @@ impl Notify {
         let mut config = dgram::Config::new();
         config.set_max_retries(self.retries);
 
+        // Prevent sending of an EDNS(0) OPT record as it causes OpenDNSSEC 2.1.14 to crash.
```

Comment:
I'm not sure that we should explicitly mention a workaround for our own software, especially that it can cause a crash. Something generic like 'some software cannot handle an OPT record in a NOTIFY' might be better.

Reply:
As we develop in the open I don't see value in being vague.

Also if using your proposed text the reader may think "I've never heard of any software that has this issue" and proceed to remove the line of code perceiving it to be useless.